### PR TITLE
Add list of verify-conformance bot test requirements

### PR DESCRIPTION
**Pull request by BobyMCbobs**
This PR adds the list of requirements that the verify-conformance bot checks for each product submission

### CODE REVIEW

**Comment by Riaankl** [comment]
@BobyMCbobs please add the `check passed` label states to the Pass column

File: instructions.md [hunk 1]
```diff
@@ -121,8 +121,25 @@ repo_url: https://github.com/yoyo.dyne/turbo-encabulator
...
+| 8   | the e2e.log output contains the Kubernetes release version                       | it seems the e2e.log does not contain the Kubernetes release version that match the submission title                                                                                                                                              | Not verifiable |      | Matching the e2e.log release version with the PR title release version                                                             |
+| 9   | the submission release version is a supported version of Kubernetes              | the Kubenetes release version in this pull request does not qualify for conformance submission anymore (https://github.com/cncf/k8s-conformance/blob/master/terms-conditions/Certified_Kubernetes_Terms.md#qualifying-offerings-and-self-testing) | Not verifiable |      | Only the current release version and two version prior is supported for conformance certification                                  |
```
Comment (Riaankl) [inline instructions.md:139]: valid conformance-product-submission

File: instructions.md [hunk 2]
```diff
@@ -121,8 +121,25 @@ repo_url: https://github.com/yoyo.dyne/turbo-encabulator
...
+| 10  | all required conformance tests in the junit_01.xml are present                   | it appears that some tests are missing from the product submission                                                                                                                                                                                | Not verifiable |      |                                                                                                                                    |
```
Comment (Riaankl) [inline instructions.md:140]: tests-verified-"+s.KubernetesReleaseVersion

File: instructions.md [hunk 3]
```diff
@@ -121,8 +121,25 @@ repo_url: https://github.com/yoyo.dyne/turbo-encabulator
...
+| 11  | all tests pass in e2e.log                                                        | it appears that some tests failed in the product submission                                                                                                                                                                                       | Not verifiable |      |                                                                                                                                    |
```
Comment (Riaankl) [inline instructions.md:141]: no-failed-tests-"+s.KubernetesReleaseVersion

File: instructions.md [hunk 4]
```diff
@@ -121,8 +121,25 @@ repo_url: https://github.com/yoyo.dyne/turbo-encabulator
...
+| 12  | the tests in junit_01.xml and e2e.log match                                      | it appears that there is a mismatch of tests in junit_01.xml and e2e.log                                                                                                                                                                          | Not verifiable |      | Verify that the junit_01.xml file and e2e.log file contain the same tests                                                          |
```
Comment (Riaankl) [inline instructions.md:143]: | Final check  |  All requirements 12 have passed for the submission!    |                                                                                                                                                                         | Not verifiable | release-"+s.KubernetesReleaseVersion
release-documents-checked    | The step will add the release+version &  release-documents-checked labels once all 12 check have passed                               |

**Review by Riaankl — COMMENTED** [review]
Table updates

**Comment by Riaankl** [comment]
/lgtm
@taylorwaggoner we added a table to explain to the end user the 12 steps being check by the CNCF-CI Bot.
If it is clear to you please merge the PR

**Comment by taylorwaggoner** [comment]
LGTM!

**Comment by BobyMCbobs** [comment]
Woohoo!
Thank you, @taylorwaggoner :tada:

**Pull request merged**